SConstruct

```diff
@@ -370,7 +370,7 @@ def scons(): # pylint: disable=too-many-locals
     sys.path.insert(0, os.path.join(Dir('#').abspath, 'utils/sl'))
     from prereq_tools import PreReqComponent
 
-    env = Environment(TOOLS=['extra', 'default'])
+    env = Environment(TOOLS=['extra', 'default', 'textfile'])
```

Comment:
What does this do?

Reply:
Add support for textfile to scons. That's required after Ken's recent changes.

Reply:
On some versions of SCons, the Substfile builder is only available if you add this tool

Reply:
I uploaded this patch: https://github.com/daos-stack/daos/pull/4388

I figured since at least two people have hit this since you did, it would be good to land it.